extensions/expression-language/Expression.cpp

```diff
@@ -636,7 +636,7 @@ Value expr_toDate(const std::vector<Value> &args) {
   auto utc = date::locate_zone("UTC");
   auto utct = date::make_zoned(utc, t);
   auto zt = date::make_zoned(zone, utct.get_local_time());
-  return Value(std::chrono::duration_cast<std::chrono::milliseconds>(zt.get_sys_time().time_since_epoch()).count());
+  return Value(int64_t{std::chrono::duration_cast<std::chrono::milliseconds>(zt.get_sys_time().time_since_epoch()).count()});
```

Comment:
With libc++, the count return value is `long long`, which is not convertible to `int64_t` (alias to `long`). `Value` doesn't have a `long long` constructor, so I had to add a non-narrowing cast to these places.